content/en/docs/concepts/cluster-administration/flow-control.md

```diff
@@ -782,6 +782,100 @@ APF adds the following two headers to each HTTP response message.
 - `X-Kubernetes-PF-PriorityLevel-UID` holds the UID of the
   PriorityLevelConfiguration object associated with that FlowSchema.
 
+## Good practices for using API Priority and Fairness
+When a given priority level exceeds its permitted concurrency, requests can 
+experience increased latency or be dropped with an HTTP 429 (Too Many Requests) 
+error. To prevent these side effects of APF, you can modify your workload or 
+tweak your APF settings to ensure there are sufficient seats available to serve
+your requests.
+
+### Workload modifications {#good-practice-workload-modifications}
+To prevent requests from queuing and adding latency or being dropped due to APF, 
+you can optimize your requests by:
+
+- Reducing the rate at which requests are executed. A fewer number of requests 
+over a fixed period will result in a fewer number of seats being needed at a 
+given time.
+- Avoid issuing a large number of expensive requests concurrently. Requests can 
+be optimized to use fewer seats or have lower latency so that these requests 
+hold those seats for a shorter duration. List requests can occupy more than 1 
+seat depending on the number of objects fetched during the request. Restricting 
+the number of objects retrieved in a list request (by using pagination or 
+limiting the request to a particular namespace) will use less total seats over a
+shorter period. Furthermore, replacing list requests with watch requests will 
+require lower total concurrency shares as watch requests only occupy 1 seat 
+during its initial burst of notifications.
+
+### Priority and fairness settings {#good-practice-apf-settings}
+You can also modify the default FlowSchema and PriorityLevelConfiguration 
+objects or create new objects of these types to better accommodate your 
+workload.
+
+APF settings can be modified to:
+- Give more seats to high priority requests.
+- Isolate non-essential or expensive requests that would starve a concurrency
+level if it was shared with other flows.
+
+#### Give more seats to high priority requests
+1. You can create a new FlowSchema which references a PriorityLevelConfiguration 
+with a larger concurrency level. This new PriorityLevelConfiguration could be an 
+existing level or a new level with its own set of nominal concurrency shares. 
+Creating a new PriorityLevelConfiguration will reduce the number of seats 
+designated for existing levels. 
+2. You can also increase the NominalConcurrencyShares for the 
```

Comment:
hm, I am not sure it's possible for default PriorityLevels - I think kube-apiserver reconciles them to default values, unfortunately.

Reply:
A user can set the apf.kubernetes.io/autoupdate-spec annotation to false to prevent these objects from being reconciled back to defaults. This is mentioned earlier in the APF doc, I'll bring it up again here for clarity.

Reply:
Ah thanks, didn't know about it :)

Reply:
What Kubernetes version is the annotation introduced in @marseel

Reply:
PR which added the annotation: https://github.com/kubernetes/kubernetes/pull/98028/commits

Checking branches with commits shows it was added in 1.22:
```
% git branch -a --contains 759a64136b0d4619d5535adb79a8367e124b06c6
* master
  remotes/origin/HEAD -> origin/master
  remotes/origin/master
  remotes/origin/release-1.22
  remotes/origin/release-1.23
  remotes/origin/release-1.24
  remotes/origin/release-1.25
  remotes/origin/release-1.26
  remotes/origin/release-1.27
  remotes/origin/release-1.28
```